folium/plugins/fullscreen.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+Fullscreen
+--------------
+
+https://github.com/brunob/leaflet.fullscreen
+
+Note: does not work from within an iframe, and because of this, does not
+work in jupyter notebook
+"""
+
+import json
+from jinja2 import Template
+
+from branca.element import MacroElement, Figure, JavascriptLink, CssLink
+from branca.utilities import none_min, none_max, iter_points
+
+
+class Fullscreen(MacroElement):
+    def __init__(self):
+        """Add button to take your Folium map fullscreen"""
+        super(Fullscreen, self).__init__()
+        self._name = 'Fullscreen'
+
+        self._template = Template("""
+        {% macro script(this, kwargs) %}
+            L.control.fullscreen().addTo({{this._parent.get_name()}});
+            {{this._parent.get_name()}}.on('enterFullscreen', function(){
+                console.log('entered fullscreen');
+            });
+
+        {% endmacro %}
+        """)  # noqa
+
+    def render(self, **kwargs):
+        super(Fullscreen, self).render()
+
+        figure = self.get_root()
+        assert isinstance(figure, Figure), ("You cannot render this Element "
+                                            "if it's not in a Figure.")
+
+        figure.header.add_children(
+            JavascriptLink(
+                "https://cdnjs.cloudflare.com/ajax/libs/"
```

Comment:
It is fine to use long lines for URLs (and more readable/copy-n-pastable :wink:). Just add a `  # noqa` comment at the end to tell the linter to skip it.